external/versions.json

```diff
@@ -1,7 +1,9 @@
 {
-    "boost": "1.81.0",
+    "boost": "1.83.0",
     "fmt": "10.1.0",
-    "qt": "6.2.7",
-    "rdkit": "2023.09.4",
+    "qt": "6.5.3",
```

Comment:
Re: qt 6.5.5 not being tagged: I guess it's probably in the commercial-only stage. It'll probably become community-accessible when there's another version that supersedes it?comm